bot.js

```diff
@@ -346,17 +467,21 @@ const commands = {
         text: 'Prints the bot\'s log'
     },
     thumbnail: {
-        fn: (msg, rawArgs, skill) => {
-            if (validSkills.has(skill)) {
-                sendUpdateMessage(msg.channel, 'Here is the thumbnail', skill, {
-                    title: skill
+        fn: (msg, rawArgs, name) => {
+            if (validSkills.has(name)) {
+                sendUpdateMessage(msg.channel, 'Here is the thumbnail', name, {
+                    title: name
+                });
+            } else if (validBosses.has(name)) {
```

Comment:
This doesn't work because the `name` arg is cast to lower case when the command args are parsed, whereas the set contains the capitalized names. Not sure what the best approach here is tbh